homeassistant/components/ads/sensor.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, ads_hub, ads_var, ads_type, name, unit_of_measurement,
         """Initialize AdsSensor entity."""
         self._ads_hub = ads_hub
         self._name = name
+        self._unique_id = ads_var
```

Comment:
Will a user never configure the same adsvar but different adstypes to make combinations of config settings?

Reply:
No, the user needs to choose the proper data type (adstype) of the TwinCAT variable (adsvar). Other types would make no sense.

Reply:
Great!